job_bundles/README.md

```diff
@@ -0,0 +1,309 @@
+rather than the alternative of developing solely an Open Job Description Job Template and using the 
+[deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html) directly. 
+By using a Job Bundle with the Deadline Cloud CLI you can easily make use of Deadline Cloud's Job Attachments feature,
+and an intuitive graphical Job submission interface. For example, after 
+[configuring the Deadline Cloud CLI](https://github.com/aws-deadline/deadline-cloud/blob/mainline/README.md#configuration) 
+you can run it with the Blender sample in this samples directory (`deadline bundle gui-submit blender_render/`) to see:
+
+![UI Shared Settings](../.images/blender_submit_shared_settings.png) ![UI Job Settings](../.images/blender_submit_job_settings.png) ![UI Job Attachments](../.images/blender_submit_job_attachments.png) 
+
+where the contents of the Job-specific settings panel are automatically generated using the `userInterface` properties of Job Parameters
+[defined in the Job Template](https://github.com/aws-deadline/deadline-cloud-samples/blob/bdd5ff5ea29eb7457c9a78ba39166b891b79151e/job_bundles/blender_render/template.yaml#L11-L19) within 
+your Job Bundle, and the [Queue Environments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-queue-environment.html)
+defined on the Queue that you are submitting to.
+
+If you prefer an alternative to a UI-based workflow for your Job Bundles, then you can submit this Job Bundle with the command
+`deadline bundle submit --name Demo -p BlenderSceneFile=<location-of-your-scene-file> -p OutputDir=<file-path-for-job-outputs> blender_render/`
+or use the `deadline.client.api.create_job_from_job_bundle` function in the [`deadline` Python package](https://github.com/aws-deadline/deadline-cloud).
+
+All of the Job submitter plugins that have been developed by the AWS Deadline Cloud team, such as the 
+[Autodesk Maya plugin](https://github.com/aws-deadline/deadline-cloud-for-maya), are simply generating a Job Bundle for your
+Job submission and then using the [Deadline Cloud Python package](https://github.com/aws-deadline/deadline-cloud) to
+submit your Job to Deadline Cloud. You can see the Job Bundles that are
+submitted by looking in the job history directory on your workstation after submitting a Job using a submitter plugin or the Deadline
+Cloud CLI. You can find your job history directory by running the command: `deadline config get settings.job_history_dir`.
+
+## Elements of a Job Bundle
+
+A Job Bundle is a directory structure that contains at least an
+[Open Job Description Job Template](https://github.com/OpenJobDescription/openjd-specifications/wiki) file, but may contain
+other files such as:
+
+```
+/template.yaml (or template.json)
+/asset_references.yaml (or asset_references.json)
+/parameter_values.yaml (or parameter_values.json)
```

Comment:
I think it'd be worth making explicit somewhere that job bundles can be used both to define a reusable workflow (e.g. our sample bundles) and to bake/archive/freeze a particular job submission (e.g. `~/.deadline/job_history`). Someone might be confused what asset references are doing in a reusable job definition.